scripts/docker/content/set_env.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
```

Comment:
Similar comments as this [PR](https://github.com/Azure/pcs-remote-monitoring-webui/pull/1235)

Reply:
Addressed the comments in this file as well